extra-rocm/rocm-device-libs/autobuild/prepare

```diff
@@ -0,0 +1 @@
+export PATH="/opt/rocm/bin:$PATH"
```

Comment:
Could be worth an `abinfo` here.

Reply:
I don't think setting a environment variable is worth abinfo.

Maybe commenting is okay, but it's meaningless to show this to build log.